### ISSUE TITLE
discover() should produce a list of the same order everytime

### ISSUE THREAD

**Issue by tilltnet** [tfeldmann/python-raumfeld#6]
If there are several listening zones, the order of the entries in the list that discover() produces is random. Sorting the list alongside UDNs or the Name should fix this.

**Comment by tfeldmann** [comment]
Maybe it's a good idea to return the devices in a dictionary of lists? Something like

``` python
result {
  "zone 1": [device1, device2],  # devices are sorted by name
  "zone 2": [device3]
} 
```

I can implement and test this along with #1 as soon as my Stereo Cubes arrive :)

**Comment by tilltnet** [comment]
Every 'Virtual Media Player' is actually one listening zone, so it might suffice to do list.sort() devices somehow. But if you plan on creating your own listening zones later on, the dictionary approach might be worthwhile.

Have fun with the cubes! :)

**Comment by tilltnet** [comment]
Adding 

```
     devices = sorted(devices, key = lambda device: device.friendly_name)
```

after line 60 eliminates the problem. I managed to integrate a "zone switch feature" in my remote with that.

**Comment by tfeldmann** [comment]
Implemented in 4bb9de079baa07eb6e425ba1ec7c7749386dfecd
